src/content/docs/en/guides/images.mdx

```diff
@@ -497,7 +497,7 @@ export default defineConfig({
 
 Image CDNs work with all Astro image options. Use an image's full URL as the `src` attribute in the `<Image />` component, an `<img>` tag, or in Markdown notation. For image optimization with remote images, also [configure your authorized domains or URL patterns](#authorizing-remote-images).
 
-Alternatively, if the CDN provides a Node.js SDK, you can use that in your project. For example, [Cloudinary’s SDK](https://cloudinary.com/documentation/node_integration) can generate an `<img>` tag with the appropriate `src` for you.
+Alternatively, the CDN may provide it's own SDKs to more easily integrate in an Astro project. For example, Cloudinary supports an [Astro SDK](https://astro.cloudinary.dev/) which allows you to easily drop in images with their CldImage component or a [Node.js SDK](https://cloudinary.com/documentation/node_integration) that can generate URLs to use with an `<img>` tag in a Node.js environment.
```

Comment:
this feels a little Cloudinary heavy relative to the section that it's being described in, but the original intent was Node.js focused and I wanted to offer the Astro SDK as an alternative option now that it's available